lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -387,7 +387,7 @@ def translate_treatments
             'endDate' => approximate_date(treatment['treatmentDateRange']['to']),
             'treatedDisabilityNames' => treatment['treatedDisabilityNames'],
             'center' => {
-              'name' => treatment['treatmentCenterName']
+              'name' => treatment['treatmentCenterName'].gsub(/[^a-zA-Z0-9 .()#&'"-]+/) { '' }.strip.to_s
```

Comment:
```suggestion
              'name' => treatment['treatmentCenterName'].gsub(/[^a-zA-Z0-9 .()#&'"-]+/, '').strip
```

I think the `to_s` is harmless but unnecessary.

Also, the more I look at EVSS' regex, the less I understand their intent, but your regex will strip out the correct characters so 🤷‍♀️

Reply:
Ah, yeah that returns a string.  When I was playing around with other regex methods, one of them didn't return a string.  I guess I didn't recheck the return type when I settled on gsub. Thanks!

Reply:
About the original EVSS regex, yeah, that thing is nearly illegible. I have no idea what their intent is beyond stripping those characters. The implication that they're looping through the two different capture groups makes my head hurt.  I think if anything, our regex will be stricter then theirs, which is preferable.